### ISSUE TITLE
Enter not exiting goldtree

### ISSUE THREAD

**Issue by wicksand420** [XorTroll/Goldleaf#157]
After installing games with goldtree, it states that you can close goldtree by pressing enter, but pressing enter does nothing. I'm on a windows 7 64bit laptop.

**Comment by CVFireDragon** [comment]
^ Same for me. I'm on Windows 10 64Bit Desktop

**Comment by Niekvdm** [comment]
The written code is valid and should be working even with applied modifiers such as CTRL and SHIFT.

Perhaps it could be writtern as the following:
```C#
// Intercept input
ConsoleKeyInfo ki = Console.ReadKey(true);
if (ki.Key.Equals(ConsoleKey.Enter))
{
    // Send finish command to goldleaf
    USB?.Write(new Command(CommandId.Finish));
    Environment.Exit(0);
}
else
{
    goto usbnsp;
}
```

**Comment by lookye** [comment]
This should be fixed with this PR #166
